online-devops-dojo/devops-kaizen/assets/dialog2.yaml

```
@@ -0,0 +1,52 @@
+---
+speaking: Santhosh
+text: >
+    Welcome to our first DevOps Kaizen event.
+---
+speaking: Dan
+text: >
+    Thanks Santhosh, how are we to approach this session?
+---
+speaking: Santhosh
+text: >
+    We know Improvements are needed, and that we can't change everything at the same time so we need to identify some focus areas based on the VSM.
+---
+speaking: Selma
+text: >
+    I have been proposing a Shift Left on Security for some time, can that be one?
+---
+speaking: Tina
+text: >
+    Dan and I have been discussing increasing the use of test automation, can we look at that?
+---
+speaking: Adam
+text: >
+    Tina and Selma, I think both of those are needed and what's more I think they would fit nicely with some ideas I have around Source Code Management, Continuous Integration and Continuous Deployment.
+---
+speaking: Chun
+text: >
+    Guys, guys, great ideas but we can't boil the ocean. Can we look at these suggestions, prioritize them based on expected value and time to implement with a view to incorporating them in to an Improvement Theme?
+---
+speaking: Dan
+text: >
+    I think so, especially as they are all inter-related, as Adam pointed out. They all will help facilitate flow and eliminate waste in our development process.
+---
+speaking: Chun
+text: >
+    Exactly! Automated testing, automated security scans, version control, and CI/CD will allow problems to be identified earlier and feedback amplified as per the three ways of DevOps.
+---
+speaking: Paulo
+text: >
+    Agree, but think we also need to be mindful of the metrics we collected during the VSM exercise lead time, processing time, and %C/A when prioritizing the next steps for the Improvement Theme.
+---
+speaking: Santhosh
+text: >
+    I agree, being able to track progress against key metrics will help us quantify the progress we make and prioritize items for inclusion in the themes.
+---
+speaking: Tina
+text: >
+    We may need more than one Improvement Theme given the breadth and scale of the issues we are trying to address.
+---
+speaking: Paulo
+text: >
+    That is fine, after all the resulting work will eventually be tracked in our Agile tool under the DevOps Kaizen epic we created. That will give us the holistic view of the improvements underway.
```

```suggestion
    That is fine, after all the resulting work will eventually be tracked in our Agile tool under the DevOps Kaizen epic we created. That will give us the holistic view of the improvements underway.
```

This is nuanced, but wanted to ask: ideally when creating an epic in this context, it refers to the OUTPUT of the kaizen recommendations, yes?  An epic called DevOps Kaizen might represent the work or velocity to RUN a kaizen event, but would ideally not be the output as an epic should read as if it is DRIVING a specific business value. so if the kaizen event created a note that we don't have CI, the resulting improvement epic would be "Create a CI/CD pipeline that runs one unit test" or something.  Given that, and again this is nuanced, we just want to ensure the student doesn't see this recommendation as creating an epic called "DevOps Kaizen epic"?  what do you think, and open to opinions as i gather the intent was likely more generic in nature but it could be interpreted different ways.

---

Agree with the approach suggested, indeed the name specified is intended to be illustrative of the name of the epic(s) to be created following a kaizen event